recipes/assimp/5.0.x/patches/0001-unvendor-deps.patch

```diff
@@ -0,0 +1,176 @@
+--- a/code/3MF/D3MFExporter.cpp
++++ b/code/3MF/D3MFExporter.cpp
+@@ -58,7 +58,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ #ifdef ASSIMP_USE_HUNTER
+ #  include <zip/zip.h>
+ #else
+-#  include <contrib/zip/src/zip.h>
++#  include <zip/zip.h>
+ #endif
+ 
+ namespace Assimp {
+--- a/code/Blender/BlenderTessellator.h
++++ b/code/Blender/BlenderTessellator.h
+@@ -147,7 +147,7 @@ namespace Assimp
+ #ifdef ASSIMP_USE_HUNTER
+ #  include <poly2tri/poly2tri.h>
+ #else
+-#  include "../contrib/poly2tri/poly2tri/poly2tri.h"
++#  include <poly2tri/poly2tri.h>
+ #endif
+ 
+ namespace Assimp
+--- a/code/CMakeLists.txt
++++ b/code/CMakeLists.txt
+@@ -893,7 +893,7 @@ ENDIF(HUNTER_ENABLED)
+ IF(HUNTER_ENABLED)
+   hunter_add_package(poly2tri)
+   find_package(poly2tri CONFIG REQUIRED)
+-ELSE(HUNTER_ENABLED)
++ELSEIF(0)
```

Comment:
You set this to false in the recipe is this still required?

Reply:
yes